modules/preview-react/form-field/lib/hooks/useFormFieldModel.tsx

```diff
@@ -6,7 +6,8 @@ export const useFormFieldModel = createModelHook({
      * Optional flag to denote if this field has an error to display. When true the `FormField.Input` will have
      * `required` set to true, and usually some subcomponents will have a error color applied.
      */
-    hasError: false,
+    error: '' as 'error' | 'alert',
```

Comment:
@NicholasBoll how can I type this better? I didn't want to reply on a consumer importing `ErrorType`, but rather a union of these two strings.

Reply:
The type is lying here. It looks like you can have `'error' | 'alert' | undefined`. If that's true:

```suggestion
    error: undefined as undefined | 'error' | 'alert',
```